fibonacci_number/FibonacciNumber.java

```diff
@@ -0,0 +1,27 @@
+import java.util.*;
+/*
+ *Fibonacci series 1,1,2,3,5,8,13,21...
+ *Find the nth value of series.
+ */
+public class FibonacciNumber {
+	public static int fibonacciNumber(int n) {
+		int sum = 0;
+		int b = 0;
+		int a = 1;		
+        if (n == 1){
+        	return 1;
+        }
+		for (int i = 1;i < n;i++){
+			sum = a + b;
+			b = a;
+			a = sum;	
+		}
+		return sum;
+	}
+
+	public static void main(String[] args) {
+		Scanner in = new Scanner(System.in);
+		int n = in.nextInt();
+		System.out.println(fibonacciNumber(n));
```

Comment:
Also user should be able to give its own, initial conditions of 1st and second number, if not provided.
You can do it this way https://stackoverflow.com/a/23093360